android/lib/src/main/java/com/ern/api/impl/core/ElectrodeBaseFragmentDelegate.java

```diff
@@ -131,7 +131,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
             rootView = mMiniAppView;
         }
 
-        setDisplayHomeAsUpEnabled(mFragment.getArguments().getBoolean(ActivityDelegateConstants.KEY_MINI_APP_FRAGMENT_SHOW_UP_ENABLED, false));
+        if(mFragment.getArguments() != null) {
```

Comment:
Missing space after `if` keyword. Also: `getArguments()` is now called four times (null checked twice) in this method.

How about something like this on top?

```java
Bundle args = mFragment.getArguments();
if (args != null) {
    // ...
}
```

Reply:
Thanks. Fixed here: https://github.com/electrode-io/ernnavigation-api-impl-native/pull/71